pokemonterminal/scripter.py

```diff
@@ -71,7 +71,7 @@ def __init_wallpaper_provider():
 
 def clear_terminal():
     # clear any updates to the terminal window's title bar
-    sys.stdout.write("\x1b]2;\x07")
+    sys.stdout.write("\033]2;\007")
```

Comment:
FYI this is octal, not decimal.

Reply:
lol true 😛 I can amend the commit message if that's something we really care about